backend/src/scrapers/linkedin.py

```
@@ -0,0 +1,373 @@
+                scraper=self.__class__.__name__,
+                query=query,
+                location=location,
+            ).warning("No LinkedIn job cards found before timeout")
+            return []
+        except Exception as exc:
+            logger.bind(scraper=self.__class__.__name__, error=str(exc)).error(
+                "Failed to load LinkedIn jobs search page"
+            )
+            raise LinkedInTransientError(
+                "Failed to load LinkedIn jobs search page"
+            ) from exc
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)